src/figuresClasses.ts

```diff
@@ -1,19 +1,69 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  constructor(
+    public color: 'red' | 'green' | 'blue',
+    public a: number,
```

Comment:
You could solve task without making public fields, as you should pass instance of class into getInfo() function, not the entire class.